data_analysis/agora/notebooks/preprocessing/preprocessing_utils.py

```diff
@@ -336,6 +336,39 @@ def _extract_ensembl_ids(
     return list(set(file_ensembl_ids))
 
 
+def load_file_with_name(
+    file_name: str, config_filename: str, token: str = None
+) -> pd.DataFrame:
```

Comment:
NIT: Update docstring to indicate this can also return `None` and why that might happen

Reply:
Done!